packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -387,6 +387,7 @@ describe('NetworkController', () => {
                 "nativeCurrency": "ETH",
                 "rpcEndpoints": Array [
                   Object {
+                    "failoverUrls": Array [],
```

Comment:
There are a few kinds of changes being made to these tests:

- For tests which do not create a new NetworkController with a specific set of network configurations, we need to now verify that Infura RPC endpoints use an empty list of failover URLs.
- For tests which _do_ create a new NetworkController with a specific set of network configurations, particularly if we are mocking `createNetworkClient` and testing the presence of network configurations in state, we pass in an explicit set of failover RPC endpoints so we can use it for comparison. I guess in most cases we don't need to care about this, it's a lot of extra information, but that's how the tests are written.